src/create.cpp

```diff
@@ -72,27 +74,25 @@ auto createFromRoot( std::string_view root_, std::string_view indexLocation, boo
 				continue;
 			}
 
-			if ( !archiveExclusionREs.empty() && matchPath( pathRel, archiveExclusionREs ) ) {
-				continue;
-			}
-
-			if ( !archiveExclusionREs.empty() && !matchPath( pathRel, archiveInclusionREs ) ) {
-				continue;
+			// check if the path should be excluded, and if it is, also check if it has been forcibly included,
```

Comment:
This is wrong, inclusions are not meant to be overrides over exclusions. If inclusions are not empty they should always be considered, and files not matching the regexes should be ignored